tests/processor/test_tasks.py

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""Test the celery functions."""
+
+from unittest.mock import Mock
+
+from masu.processor.tasks import test
+from tests import MasuTestCase
+
+
+class TaskTest(MasuTestCase):
+    """Test Cases for the Processor functions."""
+
+    def test_test_task(self):
```

Comment:
This is where I think we should be testing ready()

If we make the test task be some predefined long running task (10 seconds) then the test could be

```
ret = test('foo')
time.sleep(5)
self.assertIsFalse(ret.ready())
time.sleep(10)
self.assertIsTrue(ret.ready())
```